src/styles/globalStyles.scss

```diff
@@ -34,8 +34,12 @@ body {
 
 img {
 	-webkit-user-drag: none;
```

Comment:
why -webkit when you have autoprefixer

Reply:
this style doesn't work without `-webkit`